chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -0,0 +1,218 @@
+# ChainIndexer Documentation for RPC Providers
```

Comment:
this file should probably be in documentation/en/, also note the extension `.MD` -> `.md`